### ISSUE TITLE
reason for 800 starting timestep

### ISSUE THREAD

**Issue by shredder67** [ExplainableML/ReNO#10]
hello, in code I found specific condition for `is_hyper` used in order to set starting timestep to 800. What is the motivation for this? afaik in hyper-sd they require to use DDIM scheduler with `trailing` timesteps, meaning T=1000 is supposed to be included

**Comment by sgk98** [comment]
Hello! Thanks for your interest in our work. 
With HyperSD, we are using the 1-step SDXL U-Net checkpoint, where the default [instructions](https://huggingface.co/ByteDance/Hyper-SD) are to use the LCMScheduler instead of DDIM along with setting start timesteps to 800 since that seems to lead to better results. Feel free to modify these settings if the default 1-step images are generated alright since the noise optimization is dependent on it. 
```
###Instructions from https://huggingface.co/ByteDance/Hyper-SD
import torch
from diffusers import DiffusionPipeline, UNet2DConditionModel, LCMScheduler
from huggingface_hub import hf_hub_download
from safetensors.torch import load_file
base_model_id = "stabilityai/stable-diffusion-xl-base-1.0"
repo_name = "ByteDance/Hyper-SD"
ckpt_name = "Hyper-SDXL-1step-Unet.safetensors"
# Load model.
unet = UNet2DConditionModel.from_config(base_model_id, subfolder="unet").to("cuda", torch.float16)
unet.load_state_dict(load_file(hf_hub_download(repo_name, ckpt_name), device="cuda"))
pipe = DiffusionPipeline.from_pretrained(base_model_id, unet=unet, torch_dtype=torch.float16, variant="fp16").to("cuda")
# Use LCM scheduler instead of ddim scheduler to support specific timestep number inputs
pipe.scheduler = LCMScheduler.from_config(pipe.scheduler.config)
# Set start timesteps to 800 in the one-step inference to get better results
prompt="a photo of a cat"
image=pipe(prompt=prompt, num_inference_steps=1, guidance_scale=0, timesteps=[800]).images[0]
```

**Comment by shredder67** [comment]
Ah, I understand, I was using unified Lora checkpoint instead, `Hyper-SDXL-1step-lora.safetensors`, hence my results were significantly worse when trying to sample from `T=800`. Thank you very much!
